src/main/java/org/folio/search/service/setter/instance/SortTitleProcessor.java

```diff
@@ -0,0 +1,20 @@
+package org.folio.search.service.setter.instance;
+
+import static org.apache.commons.lang3.StringUtils.defaultIfBlank;
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
+
+import lombok.RequiredArgsConstructor;
+import org.folio.search.domain.dto.Instance;
+import org.folio.search.service.setter.FieldProcessor;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public final class SortTitleProcessor implements FieldProcessor<Instance, String> {
```

Comment:
Maybe name it as TreatEmptyAsNullProcessor to allow usage of it in another cases?

Reply:
All mod-search processors have the same naming convention, I guess, we can create an abstract class if this behaviour will repeat again, but not now